pandas/tests/io/json/test_readlines.py

```diff
@@ -27,6 +28,37 @@ def test_read_jsonl():
     tm.assert_frame_equal(result, expected)
 
 
+def test_read_jsonl_engine_pyarrow(json_dir_path):
```

Comment:
Can you create a fixture for the engine keyword and add it to all of the existing JSON tests?

Reply:
Will do👍

Reply:
@WillAyd This is the last thing I'm working on out of all the follow-ups. Work has been very busy so haven't been able to push the updates but will push something soon.